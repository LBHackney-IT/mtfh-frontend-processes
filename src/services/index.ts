export { default as locale } from "./locale";
export { default as menu } from "./menu";
export * as processes from "./processes";
export * from "./config";
