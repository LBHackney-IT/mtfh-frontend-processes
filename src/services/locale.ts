import reactHtmlParser from "react-html-parser";

import { AssetAddress } from "@mtfh/common/lib/api/asset/v1";

const locale = {
  backButton: "Back",
  bookAppointment: "Book Appointment",
  next: "Next",
  cancel: "Cancel",
  cancelProcess: "Cancel process",
  change: "Change",
  views: {
    soleToJoint: {
      title: "Sole tenant requests a joint tenure",
      steps: {
        selectTenant: "Select person",
        checkEligibility: "Eligibility Checks",
        supportingDocuments: "Supporting documents",
        finish: "Finish",
        breachOfTenancy: "Breach of tenure check",
        requestDocuments: "Request Documents",
        reviewDocuments: "Review Documents",
        submitCase: "Submit case",
      },
      actions: {
        reassignCase: "Reassign case",
        cancelProcess: "Cancel process",
        caseActivityHistory: "Case activity history",
      },
    },
    selectTenants: {
      selectTenantHint:
        "This person will be asked for proof of relationship e.g. marriage or civil partnership certificate.",
      selectTenantLabel: "Who do you want to add as a joint tenant?",
      addToTenureText: "If the person you want to add is not listed you must first",
      addToTenureLink: "add them to the tenure",
      noHouseholdMembersOver18:
        "There are no household members over the age of 18 associated to this.",
    },
    processesMenu: {
      selectProcessDescription: () =>
        reactHtmlParser(
          "<p>Start a process by selecting a process category below. <br /> Where a process is not yet supported in <strong>Manage My Home</strong> these links will open the appropriate interim Google form.</p>",
        ),
    },
    checkEligibility: {
      autoCheckIntro:
        "Manage My Home has used tenure, person and property information to automatically check eligibility for this application.",
      passedChecks: "Passed automatic eligibilty checks",
      failedChecks: "Not eligible for a sole to joint tenure",
    },
    breachOfTenancy: {
      autoCheckIntro:
        "Manage My Home has used tenure, person and property information to automatically check eligibility for this application.",
      passedChecks: "Eligibility checks passed",
      failedChecks: "Not eligible for a sole to joint tenure",
    },
    reviewDocuments: {
      passedChecks: "Eligibility checks passed",
      documentsRequested: "Documents requested",
      checkSupportingDocumentsAppointment:
        "I need to make an appointment to check supporting documents",
      appointmentScheduled: "Office appointment scheduled",
      seenPhotographicId:
        "I confirm that an outcome letter has been sent to the resident",
      seenSecondId:
        "I confirm I have seen a second form of ID (does not have to be photographic)",
      isNotInImmigrationControl:
        "I confirm that the prospective tenant is not subject to immigration control under the Asylum And Immigration Act 1996",
      seenProofOfRelationship:
        "I confirm I have seen proof of relationship to the existing tenant",
      incomingTenantLivingInProperty:
        "I confirm I have seen 3 separate documents proving the proposed tenant has been living at the property for a minimum of 12 months",
    },
  },
  errors: {
    unableToFetchRecord: "There was a problem retrieving the record",
    unableToFetchRecordDescription:
      "Please try again. If the issue persists, please contact support.",
    unableToFindProcess: "Invalid Process",
    unableToFindProcessDescription: "We have been unable to find a process of this type.",
    unableToFindState: "Invalid State",
    unableToFindStateDescription: "We have been unable to process a state of this type.",
  },
  title: "Processes",
  loadingText: "Submitting...",
  components: {
    startProcess: {
      buttonLabel: "Start process",
      thirdPartyHeading: "Sharing information with third parties",
    },
    entitySummary: {
      tenurePaymentRef: "Tenure payment ref",
      address: (assetAddress: AssetAddress): string => {
        const {
          postPreamble,
          addressLine1,
          addressLine2,
          addressLine3,
          addressLine4,
          postCode,
        } = assetAddress;

        return [
          postPreamble,
          addressLine1,
          addressLine2,
          addressLine3,

          addressLine4,
          postCode,
        ]
          .filter((addressLine) => !!addressLine)
          .join(" ");
      },
    },
  },
};

export default locale;
