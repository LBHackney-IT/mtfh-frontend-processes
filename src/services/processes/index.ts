export * from "./sole-to-joint";
export * from "./change-of-name";
