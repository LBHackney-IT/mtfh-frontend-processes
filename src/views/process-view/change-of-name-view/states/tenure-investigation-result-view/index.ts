export * from "./tenure-investigation-result-view";
