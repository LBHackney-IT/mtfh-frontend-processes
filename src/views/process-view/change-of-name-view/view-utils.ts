import { locale, processes } from "../../../services";

const { reviewDocuments } = locale.views;

const { states } = processes.changeofname;

const { interviewScheduled, interviewRescheduled, hoApprovalFailed, hoApprovalPassed } =
  states;

export const changeOfNameDocuments = [
  reviewDocuments.marriageCertificate,
  reviewDocuments.civilPartnershipCertificate,
  reviewDocuments.decreeAbsolute,
  reviewDocuments.finalOrder,
  reviewDocuments.deedPollDocument,
  reviewDocuments.statuoryDeclaration,
];

export const cancelButtonStates = [
  interviewScheduled.state,
  interviewRescheduled.state,
  hoApprovalFailed.state,
  hoApprovalPassed.state,
];
