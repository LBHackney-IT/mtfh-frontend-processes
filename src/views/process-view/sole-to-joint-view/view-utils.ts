import { processes } from "../../../services";

const { states } = processes.soletojoint;

const {
  documentsRequestedDes,
  documentsRequestedAppointment,
  documentsAppointmentRescheduled,
} = states;

export const reviewDocumentsStates = [
  documentsRequestedDes.state,
  documentsRequestedAppointment.state,
  documentsAppointmentRescheduled.state,
];
