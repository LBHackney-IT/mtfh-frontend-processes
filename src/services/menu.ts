import * as processes from "./processes";

import { Asset } from "@mtfh/common/lib/api/asset/v1/types";
import { Tenure } from "@mtfh/common/lib/api/tenure/v1/types";

interface MenuProps {
  label: string;
  link: string;
  processes?: { label: string; link: string; targetType: string }[];
  getPplQuery?: (context: any) => string;
}

const getProcessLink = (processName: string) => {
  const process = processes[processName];
  return {
    targetType: process.targetType,
    label: process.title,
    link: `/processes/${process.processName}/start`,
  };
};

const getPplQueryForRequestALegalReferral = ({ tenure, asset }): string => {
  if (!tenure || !asset) {
    return "";
  }
  const {
    householdMembers,
    tenuredAsset: { propertyReference },
  } = tenure as Tenure;
  const {
    assetAddress: { addressLine1, postCode },
  } = asset as Asset;
  const tenantNames = householdMembers
    .filter((member) => member.isResponsible)
    .map((member) => member.fullName)
    .join(", ");
  const address = addressLine1;
  const query = `usp=pp_url&entry.1699199605=${tenantNames}&entry.2086190845=${address}&entry.1956036523=${postCode}&entry.1140527432=${propertyReference}`;
  const encodedQuery = encodeURI(query);
  return encodedQuery;
};

const getPplQueryTenancyChange = ({ tenure }): string => {
  if (!tenure) {
    return "";
  }
  const {
    householdMembers,
    paymentReference,
    tenuredAsset: { fullAddress, propertyReference },
  } = tenure as Tenure;

  const tenant = householdMembers.find((member) => member.isResponsible);
  const query = `usp=pp_url&entry.1942863541=${tenant?.fullName}&entry.1090481808=${fullAddress}&entry.321634337=${propertyReference}&entry.582205322=${paymentReference}`;
  const encodedQuery = encodeURI(query);
  return encodedQuery;
};

const menu: MenuProps[] = [
  {
    label: "Sole to Joint",
    link: "https://docs.google.com/forms/d/e/1FAIpQLSdgJ9DSgGI0Aj7GO1bzLbbrArPabjS8DQwmvwb9ltB-qYYESA/viewform",
    processes: [
      getProcessLink(processes.soletojoint.processName),
      getProcessLink(processes.changeOfName.processName),
    ],
  },
  {
    label: "Other tenancy changes",
    link: "https://docs.google.com/forms/d/e/1FAIpQLSdgJ9DSgGI0Aj7GO1bzLbbrArPabjS8DQwmvwb9ltB-qYYESA/viewform",
    getPplQuery: getPplQueryTenancyChange,
  },
  {
    label: "Record a tenancy / audit homecheck",
    link: "https://docs.google.com/forms/d/e/1FAIpQLSfyXscVvgCBhbBBe7ADpV6cUqJoKrrTbU0YYRB_uXS2J4AHMQ/viewform",
  },
  {
    label: "Find legal referral response",
    link: "https://docs.google.com/spreadsheets/d/1fd6FwBXvBo2DmX8qZaXUr7hX9sE6Zr7HZsPlA_OO8Hs/edit#gid=1251146140",
  },
  {
    label: "Request a legal referral",
    link: "https://docs.google.com/forms/d/e/1FAIpQLScpHsoJQgq112HIMbW06YAFpggijwQuODJM483HGbX_PMfCMw/viewform",
    getPplQuery: getPplQueryForRequestALegalReferral,
  },
  {
    label: "Record notice served",
    link: "https://docs.google.com/forms/d/e/1FAIpQLSfRSMia4FeT6WPjrmRlsLpOXyJ4tFs3Sv_O0toxUbv_FmIjKw/viewform",
  },
  {
    label: "Rehousing",
    link: "https://docs.google.com/forms/d/e/1FAIpQLScPXSs4-xxTH2N5dUwtfmEl5mpu7kxZ6mHe86Qy_rlDNu-GbA/viewform",
  },
  {
    label: "Person",
    link: "https://docs.google.com/forms/d/e/1FAIpQLScxrWFLF7M7A3B1yzY-LZFXkV0P0qv6igcQ4wEgWHQk8DaUCw/viewform",
  },
  {
    label: "Report Japanese knotweed",
    link: "https://docs.google.com/forms/d/e/1FAIpQLSceU2H-VWi2ewaEiOAkOWULVFU2DoPSen8SRPyJrmQtRpEhfQ/viewform?gxids=7628",
  },
  {
    label: "Report overgrown tree",
    link: "https://docs.google.com/forms/d/e/1FAIpQLSfLmlLraDyOfqYhdjkOfkNWWlHUUep05G5n0GN-ewizN0Fbuw/viewform?gxids=7628",
  },
  {
    label: "Report overgrown garden",
    link: "https://docs.google.com/forms/d/e/1FAIpQLSc3mOUUEQhdRiDsS0k9XDWlfGJG0CEFAC2rnC1sYmM-idt-Nw/viewform?gxids=7628",
  },
];

export default menu;
