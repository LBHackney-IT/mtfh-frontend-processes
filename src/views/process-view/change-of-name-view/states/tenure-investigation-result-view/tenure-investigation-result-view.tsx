import React, { useState } from "react";

import { locale, processes } from "../../../../../services";
import { ProcessComponentProps } from "../../../../../types";
import { HoReviewView } from "../../../shared/ho-review-view/ho-review-view";
import { DesBox } from "../../../shared/process-components";
import { TenureInvestigationRecommendationBox } from "../../../sole-to-joint-view/states/shared";
import { tenureInvestigationResultStates } from "../../view-utils";

import { Center, Spinner, StatusErrorSummary } from "@mtfh/common/lib/components";
import { useErrorCodes } from "@mtfh/common/lib/hooks";

const { views } = locale;

const processConfig = processes.changeofname;

export const TenureInvestigationResultView = ({
  process,
  mutate,
  optional,
}: ProcessComponentProps): JSX.Element => {
  const [globalError, setGlobalError] = useState<number>();
  const errorMessages = useErrorCodes();

  if (!errorMessages) {
    return (
      <Center>
        <Spinner />
      </Center>
    );
  }

  const { person } = optional;
  return (
    <div data-testid="changeofname-tenure-investigation-result">
      {globalError && (
        <StatusErrorSummary
          id="tenure-investigation-result-global-error"
          code={globalError}
        />
      )}
      <DesBox
        title={views.submitCase.supportingDocumentsApproved}
        description={views.submitCase.viewDocumentsOnDes}
      />

      <TenureInvestigationRecommendationBox
        processConfig={processConfig}
        process={process}
      />

      {tenureInvestigationResultStates.includes(process.currentState.state) && (
        <HoReviewView
          processConfig={processConfig}
          process={process}
          mutate={mutate}
          setGlobalError={setGlobalError}
          optional={{ tenant: person }}
        />
      )}
    </div>
  );
};
