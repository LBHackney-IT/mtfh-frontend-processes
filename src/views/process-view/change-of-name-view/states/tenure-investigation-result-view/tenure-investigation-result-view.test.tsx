import {
  getReferenceDataV1,
  mockPersonV1,
  mockProcessV1,
  render,
  server,
} from "@hackney/mtfh-test-utils";
import { screen, waitForElementToBeRemoved } from "@testing-library/react";

import { processes } from "../../../../../services";
import { tenureInvestigationResultStates } from "../../view-utils";
import { TenureInvestigationResultView } from "./tenure-investigation-result-view";

const options = {
  url: "/processes/changeofname/e63e68c7-84b0-3a48-b450-896e2c3d7735",
  path: "/processes/:processName/:processId",
};

const person = mockPersonV1;

describe("tenure-investigation-result-view", () => {
  beforeEach(() => {
    jest.resetModules();
  });

  describe("check snapshots for each related state", () => {
    tenureInvestigationResultStates.forEach((state) => {
      test(`it renders TenureInvestigationResultView view correctly for ${state} state`, async () => {
        server.use(getReferenceDataV1({}, 200));
        const { container } = render(
          <TenureInvestigationResultView
            processConfig={processes.changeofname}
            process={{
              ...mockProcessV1,
              currentState: {
                ...mockProcessV1.currentState,
                state,
              },
            }}
            mutate={() => {}}
            optional={{ person }}
          />,
          options,
        );
        await waitForElementToBeRemoved(screen.queryAllByText(/Loading/));
        expect(container).toMatchSnapshot();
      });
    });
  });
});
