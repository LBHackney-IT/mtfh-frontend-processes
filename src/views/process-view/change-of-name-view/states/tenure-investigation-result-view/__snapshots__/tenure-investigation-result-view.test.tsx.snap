// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tenure-investigation-result-view check snapshots for each related state it renders TenureInvestigationResultView view correctly for InterviewRescheduled state 1`] = `<div />`;

exports[`tenure-investigation-result-view check snapshots for each related state it renders TenureInvestigationResultView view correctly for InterviewScheduled state 1`] = `<div />`;

exports[`tenure-investigation-result-view check snapshots for each related state it renders TenureInvestigationResultView view correctly for TenureInvestigationFailed state 1`] = `
<div>
  <div
    data-testid="changeofname-tenure-investigation-result"
  >
    <div
      class="mtfh-box mtfh-box--success"
    >
      <div
        class="mtfh-status-heading"
      >
        <svg
          fill="none"
          height="45"
          viewBox="0 0 45 45"
          width="45"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="22.5"
            cy="22.5"
            fill="white"
            r="21.75"
            stroke="#00664F"
            stroke-width="1.5"
          />
          <path
            clip-rule="evenodd"
            d="M35 15.5127L18.775 33L11 24.6201L14.2591 21.1074L18.775 25.9746L31.7409 12L35 15.5127Z"
            fill="#00664F"
            fill-rule="evenodd"
          />
        </svg>
        <h4
          class="lbh-heading-h4 mtfh-status-heading__title"
        >
          Supporting documents approved
        </h4>
      </div>
      <div
        class="govuk-link lbh-link lbh-link--no-visited-state"
        style="margin-left: 60px; margin-top: 17.5px;"
      >
        <a
          class="govuk-link lbh-link lbh-link--no-visited-state"
          href="http://des/teams/1/dashboard/requests"
          target="_blank"
        >
          View documents on the Document Evidence Store
        </a>
      </div>
    </div>
    <div
      class="mtfh-box mtfh-box--warning"
    >
      <div
        class="mtfh-status-box"
      >
        <svg
          fill="none"
          height="45"
          viewBox="0 0 45 45"
          width="45"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="22.5"
            cy="22.5"
            fill="white"
            r="21.75"
            stroke="#BE3A34"
            stroke-width="1.5"
          />
          <path
            d="M24.2969 26.6587H20.6035L19.8315 11.8701H25.0688L24.2969 26.6587ZM19.771 31.8354C19.771 30.9878 19.998 30.347 20.4521 29.9131C20.9062 29.4792 21.5672 29.2622 22.4351 29.2622C23.2726 29.2622 23.9185 29.4842 24.3726 29.9282C24.8368 30.3722 25.0688 31.008 25.0688 31.8354C25.0688 32.6326 24.8368 33.2633 24.3726 33.7275C23.9084 34.1816 23.2625 34.4087 22.4351 34.4087C21.5874 34.4087 20.9315 34.1867 20.4673 33.7427C20.0031 33.2886 19.771 32.6528 19.771 31.8354Z"
            fill="#BE3A34"
          />
        </svg>
        <div>
          <div
            class="mtfh-status-heading__title"
          >
            <h4
              class="lbh-heading-h4"
            >
              Tenure investigator recommendation: Decline application
            </h4>
          </div>
        </div>
      </div>
    </div>
    <form
      action="#"
      novalidate=""
    >
      <fieldset
        class="govuk-form-group lbh-form-group"
        id="choice"
      >
        <legend
          class="govuk-label lbh-label"
          for="choice-field"
        >
          
          
        </legend>
        <div
          class="govuk-radios lbh-radios"
          id="choice-field"
        >
          <h4
            class="lbh-heading-h4"
            name="choice"
            style="margin-bottom: 26px;"
          >
            Next steps
          </h4>
          <div
            class="govuk-radios__item"
          >
            <input
              class="govuk-radios__input"
              id="ho-appointment"
              name="choice"
              type="radio"
              value="appointment"
            />
            <label
              class="govuk-label govuk-radios__label"
              for="ho-appointment"
            >
              Make an appointment with the applicant for an interview
            </label>
          </div>
          <div
            class="govuk-radios__item"
          >
            <input
              class="govuk-radios__input"
              id="ho-review"
              name="choice"
              type="radio"
              value="review"
            />
            <label
              class="govuk-label govuk-radios__label"
              for="ho-review"
            >
              I have passed the case to the Area Housing Manager for review and received a decision
            </label>
          </div>
        </div>
      </fieldset>
      <button
        class="govuk-button lbh-button"
        disabled=""
        style="width: 222px;"
        type="submit"
      >
        Confirm
      </button>
    </form>
  </div>
</div>
`;

exports[`tenure-investigation-result-view check snapshots for each related state it renders TenureInvestigationResultView view correctly for TenureInvestigationPassed state 1`] = `
<div>
  <div
    data-testid="changeofname-tenure-investigation-result"
  >
    <div
      class="mtfh-box mtfh-box--success"
    >
      <div
        class="mtfh-status-heading"
      >
        <svg
          fill="none"
          height="45"
          viewBox="0 0 45 45"
          width="45"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="22.5"
            cy="22.5"
            fill="white"
            r="21.75"
            stroke="#00664F"
            stroke-width="1.5"
          />
          <path
            clip-rule="evenodd"
            d="M35 15.5127L18.775 33L11 24.6201L14.2591 21.1074L18.775 25.9746L31.7409 12L35 15.5127Z"
            fill="#00664F"
            fill-rule="evenodd"
          />
        </svg>
        <h4
          class="lbh-heading-h4 mtfh-status-heading__title"
        >
          Supporting documents approved
        </h4>
      </div>
      <div
        class="govuk-link lbh-link lbh-link--no-visited-state"
        style="margin-left: 60px; margin-top: 17.5px;"
      >
        <a
          class="govuk-link lbh-link lbh-link--no-visited-state"
          href="http://des/teams/1/dashboard/requests"
          target="_blank"
        >
          View documents on the Document Evidence Store
        </a>
      </div>
    </div>
    <div
      class="mtfh-box mtfh-box--success"
    >
      <div
        class="mtfh-status-box"
      >
        <svg
          fill="none"
          height="45"
          viewBox="0 0 45 45"
          width="45"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="22.5"
            cy="22.5"
            fill="white"
            r="21.75"
            stroke="#00664F"
            stroke-width="1.5"
          />
          <path
            clip-rule="evenodd"
            d="M35 15.5127L18.775 33L11 24.6201L14.2591 21.1074L18.775 25.9746L31.7409 12L35 15.5127Z"
            fill="#00664F"
            fill-rule="evenodd"
          />
        </svg>
        <div>
          <div
            class="mtfh-status-heading__title"
          >
            <h4
              class="lbh-heading-h4"
            >
              Tenure investigator recommendation: Approve application
            </h4>
          </div>
        </div>
      </div>
    </div>
    <form
      action="#"
      novalidate=""
    >
      <fieldset
        class="govuk-form-group lbh-form-group"
        id="choice"
      >
        <legend
          class="govuk-label lbh-label"
          for="choice-field"
        >
          
          
        </legend>
        <div
          class="govuk-radios lbh-radios"
          id="choice-field"
        >
          <h4
            class="lbh-heading-h4"
            name="choice"
            style="margin-bottom: 26px;"
          >
            Next steps
          </h4>
          <div
            class="govuk-radios__item"
          >
            <input
              class="govuk-radios__input"
              id="ho-appointment"
              name="choice"
              type="radio"
              value="appointment"
            />
            <label
              class="govuk-label govuk-radios__label"
              for="ho-appointment"
            >
              Make an appointment with the applicant for an interview
            </label>
          </div>
          <div
            class="govuk-radios__item"
          >
            <input
              class="govuk-radios__input"
              id="ho-review"
              name="choice"
              type="radio"
              value="review"
            />
            <label
              class="govuk-label govuk-radios__label"
              for="ho-review"
            >
              I have passed the case to the Area Housing Manager for review and received a decision
            </label>
          </div>
        </div>
      </fieldset>
      <button
        class="govuk-button lbh-button"
        disabled=""
        style="width: 222px;"
        type="submit"
      >
        Confirm
      </button>
    </form>
  </div>
</div>
`;

exports[`tenure-investigation-result-view check snapshots for each related state it renders TenureInvestigationResultView view correctly for TenureInvestigationPassedWithInt state 1`] = `
<div>
  <div
    data-testid="changeofname-tenure-investigation-result"
  >
    <div
      class="mtfh-box mtfh-box--success"
    >
      <div
        class="mtfh-status-heading"
      >
        <svg
          fill="none"
          height="45"
          viewBox="0 0 45 45"
          width="45"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="22.5"
            cy="22.5"
            fill="white"
            r="21.75"
            stroke="#00664F"
            stroke-width="1.5"
          />
          <path
            clip-rule="evenodd"
            d="M35 15.5127L18.775 33L11 24.6201L14.2591 21.1074L18.775 25.9746L31.7409 12L35 15.5127Z"
            fill="#00664F"
            fill-rule="evenodd"
          />
        </svg>
        <h4
          class="lbh-heading-h4 mtfh-status-heading__title"
        >
          Supporting documents approved
        </h4>
      </div>
      <div
        class="govuk-link lbh-link lbh-link--no-visited-state"
        style="margin-left: 60px; margin-top: 17.5px;"
      >
        <a
          class="govuk-link lbh-link lbh-link--no-visited-state"
          href="http://des/teams/1/dashboard/requests"
          target="_blank"
        >
          View documents on the Document Evidence Store
        </a>
      </div>
    </div>
    <div
      class="mtfh-box"
    >
      <div
        class="mtfh-status-box"
      >
        <svg
          fill="none"
          height="45"
          viewBox="0 0 45 45"
          width="45"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="22.5"
            cy="22.5"
            fill="white"
            r="21.75"
            stroke="#0B0C0C"
            stroke-width="1.5"
          />
          <path
            d="M20.2251 12.7026C20.2251 11.1991 21.0627 10.4473 22.7378 10.4473C24.4129 10.4473 25.2505 11.1991 25.2505 12.7026C25.2505 13.4191 25.0386 13.9792 24.6147 14.3828C24.201 14.7764 23.5754 14.9731 22.7378 14.9731C21.0627 14.9731 20.2251 14.2163 20.2251 12.7026ZM25.0386 34H20.4219V17.0771H25.0386V34Z"
            fill="#0B0C0C"
          />
        </svg>
        <div>
          <div
            class="mtfh-status-heading__title"
          >
            <h4
              class="lbh-heading-h4"
            >
              Tenure investigator recommendation: Interview Applicant
            </h4>
          </div>
        </div>
      </div>
    </div>
    <form
      action="#"
      novalidate=""
    >
      <fieldset
        class="govuk-form-group lbh-form-group"
        id="choice"
      >
        <legend
          class="govuk-label lbh-label"
          for="choice-field"
        >
          
          
        </legend>
        <div
          class="govuk-radios lbh-radios"
          id="choice-field"
        >
          <h4
            class="lbh-heading-h4"
            name="choice"
            style="margin-bottom: 26px;"
          >
            Next steps
          </h4>
          <div
            class="govuk-radios__item"
          >
            <input
              class="govuk-radios__input"
              id="ho-appointment"
              name="choice"
              type="radio"
              value="appointment"
            />
            <label
              class="govuk-label govuk-radios__label"
              for="ho-appointment"
            >
              Make an appointment with the applicant for an interview
            </label>
          </div>
          <div
            class="govuk-radios__item"
          >
            <input
              class="govuk-radios__input"
              id="ho-review"
              name="choice"
              type="radio"
              value="review"
            />
            <label
              class="govuk-label govuk-radios__label"
              for="ho-review"
            >
              I have passed the case to the Area Housing Manager for review and received a decision
            </label>
          </div>
        </div>
      </fieldset>
      <button
        class="govuk-button lbh-button"
        disabled=""
        style="width: 222px;"
        type="submit"
      >
        Confirm
      </button>
    </form>
  </div>
</div>
`;
